collect_app/src/androidTest/java/org/odk/collect/android/tasks/InstanceUploaderTaskTest.java

```diff
@@ -57,7 +57,7 @@ public void shouldUploadAnInstance() throws Exception {
         HEAD: {
             RecordedRequest r = nextRequestFor(server);
             assertEquals("HEAD", r.getMethod());
-            assertMatches("/submission\\?deviceID=imei%3A\\d+", r.getPath());
+            assertMatches("/submission\\?deviceID=\\w+%3A\\d+", r.getPath());
```

Comment:
This still isn't working quite right for me. I think the first `/` needs to be escaped. The `?` does need to be escaped but right now there's a literal `\` instead. I believe the actual identifier can be [0-9a-zA-Z].

Something like `<\/submission\?deviceID=\w+%3A[0-9a-zA-Z]+>` is what I would expect.